.travis.yml

```diff
@@ -92,7 +92,7 @@ install:
   # Install ldoc for building docs.
   - |
     if [ "$BUILD_APIDOC" == "true" ]; then
-      travis_retry sudo luarocks install ldoc 1.4.4
+      travis_retry sudo luarocks install ldoc 1.4.6
```

Comment:
It was pinned for some workaround - mentioned in the commit message.
It can be upgraded now, but is still pinned to avoid unexpected changes.